src/SPARQLStore/Connector/FusekiHttpRepositoryConnector.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace SMW\SPARQLStore\Connector;
+
+use SMW\SPARQLStore\Exception\BadHttpDatabaseResponseException;
+use SMW\SPARQLStore\QueryEngine\RepositoryResult;
+use SMW\SPARQLStore\QueryEngine\XmlResponseParser;
+
+/**
+ * @see https://jena.apache.org/documentation/serving_data/index.html
+ *
+ * @license GNU GPL v2+
+ * @since 2.0
```

Comment:
The tags are here to help people figure out compat stuff, so a new name means it should be changed. You could extend it and have something like `@since 2.2, version with different name since 2.0`. Also, `git blame` is a thing if you want to dig deeper into it anyway